components/centraldashboard/public/components/main-page.js

```diff
@@ -1,79 +1,78 @@
-import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import {PolymerElement, html} from '@polymer/polymer/polymer-element.js'
```

Comment:
I feel pretty strongly that this is not something we should change. We should follow a commonly accepted standard and in this case I think it makes sense to choose the [Google JS Style guide](https://google.github.io/styleguide/jsguide.html#formatting-semicolons-are-required) which explicitly requires semi-colons.

There's an [eslint](https://github.com/google/eslint-config-google) configuration we can add to enforce this.

Reply:
Big plus one here.